zookeeper/experiment.py

```diff
@@ -0,0 +1,28 @@
+from typing import Callable, Optional, Union, List
+
+from tensorflow import keras
+from zookeeper.dataset import Dataset
+from zookeeper.job import Job
+from zookeeper.model import Model
+from zookeeper.preprocessing import Preprocessing
+
+
+class Experiment(Job):
+    """
+    A wrapper around a Keras experiment. Subclasses must implement their
+    training loop in `build`.
```

Comment:
Where is build being called?
```suggestion
    training loop in `run`.
```